src/amber/router/cookies/store.cr

```diff
@@ -39,7 +39,11 @@ module Amber::Router::Cookies
 
     def self.build(request, secret)
       headers = request.headers
-      host = request.host
+      host = {% if compare_versions(Crystal::VERSION, "1.0.0-0") >= 0 %}
```

Comment:
This'll need to be updated to `0.36.0-dev` given 1.0 isn't the next release anymore.  Just a heads up.